neptunecontrib/api/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
```

Comment:
You should not update copyright date in existing files. It should always be the date of first publication.

Reply:
```suggestion
# Copyright (c) 2019, Neptune Labs Sp. z o.o.
```